pkgs/daixiang0/gci/registry.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  - type: go_install
+    repo_owner: daixiang0
+    repo_name: gci
+    description: GCI, a tool that control golang package import order and make it always deterministic
+    version_source: github_tag
```

Comment:
I don't think this is needed because this tool has GitHub Relesases.

https://github.com/daixiang0/gci/releases